book/security.md

```diff
@@ -433,23 +431,18 @@ Cross-site requests
 ===================
 
 Cookies are site-specific, so one server shouldn't be sent another
-server's cookies.[^tls][^dns][^bgp][^oof] But if an attacker is
+server's cookies.[^tls] But if an attacker is
 clever, they might be able to get *the server* or *the browser* to
 help them steal cookie values.
 
 [^tls]: Well... Our connection isn't encrypted, so an attacker could
     read it from an open Wifi connection. But another *server*
-    couldn't.
-
-[^dns]: Well... Another server could hijack our DNS and redirect our
+    couldn't. Or, another server could hijack our DNS and redirect our
```

Comment:
```suggestion
    couldn't. Though, another server could hijack our DNS and redirect our
```

Reply:
Otherwise it sounds weird. Maybe "Although"?

Reply:
Does it? Seems ok to me...

Reply:
I updated the text to be phrased, better, WDYT?